src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -487,14 +482,16 @@ def process_instances_for_env(
                     .squeeze(0)
                     .bool()
                 )
+            else:
```

Comment:
@yvsriram I remember fixing this problem before, but i was getting a crash here. Is the version of this in the demo branch actually up to date? Should we replace it with the one in main?

Reply:
The if-else condition surrounding [instance_mask_downsampled](https://github.com/facebookresearch/home-robot/blob/645a338f8c77250f9fd09ea6ff007126f0dff05b/src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py#L532) was added to main [here](https://github.com/facebookresearch/home-robot/commit/aa854b613dd2740838702dc4472991304b0461f0) (was this necessary - interpolate with `du_scale=1` would have still worked right?). This commit is already present on demo but the else part might have been removed during merge.

Reply:
@yvsriram i was actually thinking we should just copy in the code from main directly